driver-core/src/test/unit/com/mongodb/ClientSessionOptionsSpecification.groovy

```diff
@@ -18,6 +18,8 @@ package com.mongodb
 
 import spock.lang.Specification
 
+import java.util.concurrent.TimeUnit
+
 class ClientSessionOptionsSpecification extends Specification {
```

Comment:
This class was also moved from the drivers-legacy dir. There seemed to be no reason to have it there as it tests driver-core functionality.